connect/test/run_tests.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 
-# install goss
+# create temporary startup log file
+cat > /tmp/startup.log
+
+# start connect
+tini -- /usr/local/bin/startup.sh 2>/tmp/startup.log &
```

Comment:
```suggestion
tini -- /usr/local/bin/startup.sh > /tmp/startup.log 2>&1 &
```

Otherwise `stdout` will still go to the terminal 😄